app/js/components/App.jsx

```diff
@@ -49,9 +49,7 @@ export default class App extends Component {
   handleError = (error) => {
     if (error.response) {
       this.toastError(error.response.data.error.message);
-      setTimeout(() => {
-        this.props.history.push('/login');
-      }, 3000);
+      this.props.history.push('/login');
```

Comment:
Why is the timeout being removed and why was it there before?

Reply:
@kmunene, the toastr at the time was in the same component and if I redirected immediately then the notification wouldn't appear. However, I created a ticket to take the toastr out of the Router scope so the setTimeout is no longer necessary.

Reply:
Okay cool.